src/QsCompiler/Tests.Compiler/SyntaxTests.fs

```diff
@@ -38,12 +38,17 @@ let ``String parser tests`` () =
             Assert.Equal(offset + 1, parsed.Length)
             Assert.Equal(char, parsed.[offset])
         | _ -> Assert.True(false, "failed to parse")
-    testChar 0 '\t' "\"\t\""
-    testChar 0 '\n' "\"\r\"" // carriage returns will get replace by \n
-    testChar 0 '\n' "\"\n\""
-    testChar 3 '\t' "$\"{0}\t\""
-    testChar 3 '\n' "$\"{0}\r\"" 
-    testChar 3 '\n' "$\"{0}\n\""
+    testChar 0 '\t' "\"\\t\""
+    testChar 0 '\r' "\"\\r\""
+    testChar 0 '\n' "\"\\n\""
+    // testChar 0 '\"' "\"\\\"\""
```

Comment:
I added test cases for all the escape characters I think we are supposed to support (based on [our docs](https://docs.microsoft.com/en-us/quantum/user-guide/language/expressions?view=qsharp-preview#string-expressions)), but had to comment out the cases for `\\` and `\"` which still fail for some reason, even though `\{` in interpolated strings works fine. Anyone have any ideas why those characters still don't work correctly?

Reply:
If you precede the string with a `@` character, it will be a verbatim string. That may help with readability here.

Reply:
Hmm... not sure what changed, but I was able to get `\"` working. I still don't know why `\\` is giving me grief.